### ISSUE TITLE
[A11y]: BMT 2 - Evidence Requests - A11y support/research

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#127372]
# 🔧 Accessibility Support Request

Some questions have come up in the Evidence Request work about the accessibility of certain designs in the mobile app - drawers & a "Call va" button.

TODO
- [x] Review [Slack conversation](https://dsva.slack.com/archives/C02M11AKKNU/p1765226356643419
) about drawers in the mobile channel 
- [x] Review[ current Figmas](https://www.figma.com/design/XzDXlx01l0m2Jris2057qN/Benefits-Evidence-Updates-%7C-VA-Mobile?node-id=5613-43612&t=K2QRdVjpODrQzby4-0) for Evidence Requests
- [x] Identify where drawers are used in the mobile app
  - 5103 drawers
  - Home → Veteran Status Card, it comes up from below and has a close button, you can still see Profile / app in the background but you can’t click any of it, focus is trapped.
  - Talk to the Veterans crisis line screen
  - Prescriptions Help screen
  - Travel help screen
  - This list is not exhaustive, I'm still auditing the app!
- [x] Test accessibility of current drawer experience
   - As long as focus is consistently managed when opening/closing the drawer I don't have any problems with going forward with drawers in the design
   - As mentioned below, be aware of the heading structure and content on screens when they are converted to drawers. Make the headings descriptive and make sure not to have duplicate heading text on pages.
- [x] See if there are any issues with a "Call VA" button experience
  - There aren't any obvious problems / technical implementation issues with the idea of having a “Call VA” button. It will be easier to click since it will have a larger target area. There might be some confusion about expected behavior because there is the “Talk to the Veterans Crisis line now” red button that is at the top of every screen. Actually, that button is really a link that opens a drawer with a bunch more links they have to navigate to in order to complete the action.
  - The “Call VA” button might also announce better. Right now the telephone links announce as “8 0 0 6 9 8 2 4 1 1 link” which is a little confusing.
  - I think you would still want to trigger the confirmation screen to actually place the call before the action happens.
- [x] See if there is any feedback about using an expandable component or use + location of a common help button for evidence request specific information [Need help exploration ](https://www.figma.com/design/XzDXlx01l0m2Jris2057qN/Benefits-Evidence-Updates-%7C-VA-Mobile?node-id=5033-30219&t=AFO02k4vmabXdSrN-4)
  - I’d recommend just having the help icon in the top right (higher than the current mock, in line with the cancel button) that opens the drawer with more information. Other notes:
      - The expandable component/accordion - I am discouraged against this for a couple of reasons. In just the example authorization screen there is so much information in the mock that adding the Need Help information in a component that needs to be opened is going to be too much for folks (especially cognitively). If a screen reader user loses their focus they’re going to have a hard time getting back to where they were. The expandable component on mobile is also not very visually clear about whether it is open or closed. The web version is better.
      - In addition to moving the help button up to be in line with “cancel” button, I’d suggest making the main heading on that drawer more descriptive, like they are in travel claims and prescriptions sections. So for example “Evidence Request Help” and “Need help with your evidence request?”
      - Make the phone number links within the drawer clickable as well.
      - In the Figma to make it more clear that the Help screen is a drawer, you can have the top of the previous screen peeking out.
- [x] See [Direct deposit display](https://www.figma.com/design/XzDXlx01l0m2Jris2057qN/Benefits-Evidence-Updates-%7C-VA-Mobile?node-id=5432-20953&t=AFO02k4vmabXdSrN-4) for any feedback about displaying [this screen ](https://www.figma.com/design/XzDXlx01l0m2Jris2057qN/Benefits-Evidence-Updates-%7C-VA-Mobile?node-id=5432-20671&t=Ghb83duKKieHcMFA-4)after selecting the button Add direct deposit information 
  -  Not a ton of concerns on this new screen (is it a drawer?), but it does stick out to me that the heading "Direct Deposit Information" appears twice, we will need to update those headings to make them unique. 

## Background
- **Team name:** BMT 2 "Bee's Knees"
- **Product name:** VAHB Mobile app
- **Feature name:** Evidence Requests
- **Contact person:** @brian-wilke 
- **Next Collaboration Cycle touchpoint & date if applicable:** Midpoint Review
- **Link to PR/Figma/prototype etc:** https://www.figma.com/design/XzDXlx01l0m2Jris2057qN/Benefits-Evidence-Updates-%7C-VA-Mobile?node-id=5613-43612&t=K2QRdVjpODrQzby4-0
- **Links to any related GitHub issues:**
- **Additional details on your request:**

## Yes, I have

- [x] Formatted the GitHub issue title to include: [A11y]: Product - Feature - Request
- [x] Set my project under "Projects"
- [x] Added the appropriate team/project label under "Labels" (e.g., Decision Reviews, BMT1, or BMT2)
- [x] Verified that @bacitracin is assigned to this ticket
